pkg/validation/validate.go

```diff
@@ -162,6 +162,10 @@ func ValidateLabels(limits LabelValidationLimits, tenantID string, ls []*typesv1
 			if err != nil {
 				return err
 			}
+			level.Debug(util.Logger).Log(
```

Comment:
Suggestions: Worth logging the service_name, to better understand where those metrics coming from.